src/js/pages/OffersList/index.jsx

```diff
@@ -86,7 +88,11 @@ class OffersList extends Component {
   };
 
   render() {
-    let filteredOffers = this.props.offers;
+    const toBN = web3.utils.toBN;
```

Comment:
You can put that `const` at the top of the file. That way we don't reinstantiate it every render